cmd/spree_cmd.gemspec

```diff
@@ -26,5 +26,5 @@ Gem::Specification.new do |s|
 
   s.add_development_dependency 'rspec'
   # Temporary hack until https://github.com/wycats/thor/issues/234 is fixed
```

Comment:
Is this comment still required?

Reply:
@poudelprakash nah, can be removed, feel free to submit a PR with this change